app/styles/ui/_tutorial.scss

```
@@ -1,12 +1,117 @@
 .tutorial-panel-component {
+  display: flex;
+  flex-flow: column nowrap;
   min-width: 300px;
+  max-width: 350px;
+  background-color: var(--background-color);
+  box-shadow: var(--base-box-shadow);
+  z-index: var(--side-panel-z-index);
+  overflow-y: scroll;
+
+  .titleArea {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: var(--spacing-triple) var(--spacing-double) var(--spacing-double) var(--spacing-quint);
+
+    h1 {
+      margin: 0;
+      font-weight: var(--font-weight-light);
```

Could we use one of the predefined font-sizes here? I believe `--font-size-lg` is the closest one to what's inferred right now.
```suggestion
      font-weight: var(--font-weight-light);
      font-size: var(--font-size-lg);
```